### PR TITLE
Upgrading NodeUnit reveals failures in w3c tests?

### PR DESCRIPTION
**Pull request by domenic**
Please help @Sebmaster! It looks like the first 9 w3c tests pass and the last 9 do not? I tried rearranging them and the same thing happens. My best guess is something wrong with w3ctest.js?

### CODE REVIEW

File: test/worker-runner.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ self.onmessage = function (e) {
 
     "living-dom/attributes.js": require("../test/living-dom/attributes.js"), // 1/11
     "living-dom/compare-document-position.js": require("../test/living-dom/compare-document-position.js"), // 0/20
+    "living-dom/event-target.js": require("../test/living-dom/event-target.js"), // 0/20
```
Comment (domenic) [inline test/worker-runner.js]: 0/20? :P

**Comment by domenic** [comment]
LGTM, with nice atomic commits too. Merge at will, or I will do so when I get a chance. Great stuff, and thanks for saving the day :D

**Comment by Sebmaster** [comment]
There might actually be another problem at the event fireing order in regards to legacy events although I think it's correct. We should at least add another test for that in any case.

**Comment by domenic** [comment]
You mean like `.onload = ...`? Yeah I think we have an open issue for that. https://github.com/tmpvar/jsdom/issues/696

File: test/worker-runner.js (hunk 1 above)
Comment (Sebmaster) [inline test/worker-runner.js]: Fixed

**Comment by Sebmaster** [comment]
Oh yeah, right. We're good then.

**Pull request merged**